packages/form/src/scss/_component.scss

```diff
@@ -314,9 +315,15 @@ $use-encapsulation: false !default;
 
     &[disabled] {
       cursor: not-allowed;
+      border-color: $form-border-color;
+      background-color: core.$uq-neutral-light80;
 
       + label {
-        opacity: .5;
+        opacity: .5; /* need to replace with a solid colour */
```

Comment:
did you want to update this to use a hex color instead of opacity?

Reply:
I have updated the Colours.